### PR TITLE
Refactor ADOPTERS list

### PR DESCRIPTION
**Pull request by mazzystr**
Signed-off-by: Chris Callegari <mazzystr@gmail.com>

kind/enhancement

**What this PR does / why we need it**:
Let's update out ADOPTERS format.  The website will devour this file then publish to the website.

### CODE REVIEW

**Comment by mazzystr** [comment]
@fabiand ping

**Comment by fabiand** [comment]
lgtm

**Review by enp0s3 — APPROVED** [review]
/approve

**Comment by mazzystr** [comment]
@fabiand @enp0s3  What do I need to do to get CI to be happy and get a merge?

File: ADOPTERS.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
-| [CoreWeave](https://www.coreweave.com) | 2020 | A Kubernetes native cloud provider with focus on GPUs at scale. KubeVirt allows us to co-locate non-containerizable workloads such as Virtual Desktops next to compute intensive containers executing on bare metal. All orchestrated via the Kubernetes API leveraging the same network policies and persistent volumes for both VM and containerized workloads. |
-| [Civo](https://www.civo.com) | 2020 | We are using KubeVirt as part of our stack to enable tenant cluster provisioning within Civo cloud. |
-| [SUSE](https://www.suse.com/) | 2020 | SUSE believes KubeVirt is the best open source way to handle Virtual Machines on Kubernetes today. We offer this additional possibility to our customers by leveraging KubeVirt in our products. |
-| [Cloudflare](https://www.cloudflare.com/) | 2018 | Cloudflare uses KubeVirt within its core data centers to accommodate use cases of our teams that are less container friendly, such as our CI runners, while still taking advantage of the Kubernetes environment. |
```
Comment (dhiller) [inline ADOPTERS.md:14]: I see Cloudflare is removed here, aren't they using KubeVirt any more?

**Comment by dhiller** [comment]
> @fabiand @enp0s3 What do I need to do to get CI to be happy and get a merge?

Just issue a `/retest` command on the PR if any checks marked as required have failed. But you wouldn't need to, retest-bot would do this for you, it would just speed up the process a bit.

**Pull request merged**